test/integration/README.adoc

```diff
@@ -0,0 +1,29 @@
+= CodeReady Containers Integration Tests
+:toc: macro
+:toc-title:
+:toclevels:
+
+toc::[]
+
+[[intro-to-integration-testing]]
+== Introduction
+Integration tests use http://github.com/code-ready/clicumber[Clicumber] package which provides basic functionality for testing CLI binaries.
+
+Clicumber allows to run commands in persistent shell instance (shell, tcsh, zsh, cmd, or powershell), assert its outputs (stdout, stderr or exitcode), check configuration files, etc.
+
+The general functionality of Clicumber is then extended by CRC specific test code to cover the whole functionality of CRC.
```

Comment:
We should be using "CodeReady Containers" in documentation, not "CRC". This sentence could also be moved to the end of the previous paragraph, as it's directly related to the contents of that paragraph.